ui/src/components/app-sharing/app-sharing.tsx

```diff
@@ -102,9 +100,9 @@ export const AppSharing = ({
   setIsPublic,
 }: AppSharingProps): React.ReactElement => {
   const [currentUser] = useRecoilState<UserState | undefined>(defaultUser);
-  const [message] = useState(
-    'Warning: adding individuals or groups will allow others to access this app.',
-  );
+  // const [message] = useState(
+  //   'Warning: adding individuals or groups will allow others to access this app.',
+  // );
```

Comment:
Remove this

Reply:
Removed 'Warning:' and put back in per conversation